README.md

```diff
@@ -707,6 +707,10 @@ You can find more examples in the `examples` directory of this repository.
 
         * **screen** - _number_ - Screen number to use **Default:** `0`
 
+        * **protocol** - _string_ - The name of the X11 authentication method used **Default:** `MIT-MAGIC-COOKIE-1`
+
+        * **cookie** - _string_ - The X11 authentication cookie encoded in hexadecimal. **Default:** random 16 byte hex string in ASCII `crypto.randomBytes(16).toString('hex')`
```

Comment:
I would just say 'The X11 authentication cookie encoded as a hexadecimal string' and just say that the default is a 'random 16 byte hexadecimal string' since the method of how it is generated is not important.

Reply:
makes sense